### ISSUE TITLE
Bug: KAlarm showing alarm window.

### ISSUE THREAD

**Issue by Oxyd76** [komh/kalarm#5]
KAlarm is open alarm window if set periodical event(every hour), uncheck "Show alarm window", and check "Play sound".
![snapshot-2016-12-06-04-41-32](https://cloud.githubusercontent.com/assets/7885175/20909896/41bb241c-bb6f-11e6-878c-f42fc9885559.png)

**Comment by komh** [comment]
Thanks. I've confirmed. An alarm window is opened unconditionally if an alarm is enabled.

However, I did this because it's not possible to stop playing sounds which loops indefinitely unless an alarm window is opened.

I'll find another ways such as loop count limit or synchronize Show and alarm window and Play sound check.

**Comment by komh** [comment]
Here is the fixed version including "Start minimized" of #4. Test, please...
[kalarm_test.zip](https://github.com/komh/kalarm/files/656331/kalarm_test.zip)

**Comment by Oxyd76** [comment]
I tried [kalarm_test.zip](https://github.com/komh/kalarm/files/656331/kalarm_test.zip). "Start minimized" feature works fine!
Bug of this issue, hmm ... When I set the "Alarm Sound" is enabled, the check box "Show an Alarm window" is automatically enabled and does not unchecked until I turn off the "Alarm Sound". Of course the same box is displayed and ... the sound plays repeatedly until pressed "Ok". In the version from Hobbes sound played once.
